code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -134,6 +152,16 @@
 		var/datum/action/A=X
 		A.UpdateButtonIcon()
 
+/obj/item/clothing/head/helmet/space/plasmaman/AltClick(mob/user)
+	..()
+	. = COMSIG_CLICK_ALT
+	if(!attached_hat)
+		return
+	user.put_in_active_hand(attached_hat)
+	to_chat(user, "<span class='notice'>You removed [attached_hat.name] from helmet!</span>")
+	attached_hat = null
+	update_icon()
```

Comment:
idk how to use github, so idk if this is cluttering and making a new review, but i tried this and it worked as maybe a quick ez solution
```suggestion
	if(user.put_in_active_hand(attached_hat))
		to_chat(user, "<span class='notice'>You removed [attached_hat.name] from helmet!</span>")
		attached_hat = null
		update_icon()
	else
		to_chat(user, "<span class='notice'>You can't remove [attached_hat.name] because something is in your hand!</span>")
```

Reply:
![image](https://user-images.githubusercontent.com/42877449/144157234-73358000-ae2d-4eae-bbac-42a43b956cee.png)

Reply:
probably should have it drop it in your other hand or on the floor, but this is ez fix

Reply:
> probably should have it drop it in your other hand or on the floor

now that i think about it, this seems like it'd be pretty easy to also do. just do the same check for if it can place it in your off hand after, and if all else fails just move to turf or whatever you're on

Reply:
for putting in other hand. dunno how to put on ground
```suggestion
	if(user.put_in_active_hand(attached_hat))
		to_chat(user, "<span class='notice'>You removed [attached_hat.name] from helmet!</span>")
		attached_hat = null
		update_icon()
	if(user.put_in_inactive_hand(attached_hat))
		to_chat(user, "<span class='notice'>You removed [attached_hat.name] from helmet!</span>")
		attached_hat = null
		update_icon()
	else
		to_chat(user, "<span class='notice'>You can't remove [attached_hat.name] because your hands are full!</span>")
```